### PR TITLE
Go back to CommonJS modules for now...

### PR DESCRIPTION
**Pull request by kevana**
...until https://phabricator.babeljs.io/T2817 is resolved.

First crack at the conversion, not entirely sure if this works but I'll verify tomorrow.

### CODE REVIEW

File: src/instrument.js [hunk 1]
```diff
@@ -342,4 +343,7 @@ export default function instrument(monitorReducer = () => null) {
     const liftedStore = createStore(liftReducer(reducer));
     return unliftStore(liftedStore, liftReducer);
   };
-}
+};
+
+module.exports.ActionTypes = ActionTypes;
```
Comment (kevana) [inline src/instrument.js]: Is this kosher for achieving a default and named exports?

**Comment by kevana** [comment]
Fixes #213

**Comment by SimenB** [comment]
@gaearon Could this be looked at? :)

@kevana Tests doesn't need to change, do they?

**Comment by kevana** [comment]
@SimenB I don't think it needs any test changes, module behavior should be the same as before. I did add exports.spec.js as a sanity check for exports. I dropped this version into my apps and didn't see any issues but I'm still not completely confident that I got the default and named exports correct.

**Comment by gaearon** [comment]
Thanks, but I think I won't support DevTools on IE8.
Even Microsoft [ends support for IE < 11 tomorrow](https://www.microsoft.com/en-us/WindowsForBusiness/End-of-IE-support).
You can use something like `es3ify` as a global transformation on your project if you need it.

**Comment by kevana** [comment]
I understand completely, it's an easy fix with webpack. Hopefully the really long tail of IE8 traffic starts dying a bit faster...

**Comment by SimenB** [comment]
IE8 will still be supported in enterprise, so my project is stuck with it for now...

In this case we can just use es3ify though, so no real problem. As long as no getters are used, I'm happy :smile:

**Comment by gaearon** [comment]
Microsoft is not going to support it for anybody, not even enterprise customers. If some enterprises still decide to use unsupported version without security fixes at this point it's their problem.

**Comment by kevana** [comment]
The security fixes are their problem but the lost revenue is my problem. It's hard for me to make a business case for dropping support when a material percentage of our customers still use it. Hopefully this pushes the stragglers to finally upgrade.

**Comment by gaearon** [comment]
It's a pain for library authors and it's gotta go at some point. I understand your concerns but JS libraries are going this way now that MS doesn't support it, and at this point the only solution we can offer is transpile to ES3 by yourself or use older versions of the tools. Cheers.

**Pull request closed without merging**